asv/__init__.py

```diff
@@ -3,7 +3,7 @@
 
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-
+                        
```

Comment:
I think flake8 should be complaining about this. Are you running flake8 at all to check your changes?